src/functions-reference/transform_functions.qmd

```diff
@@ -0,0 +1,537 @@
+a `K` by `K` Cholesky factor of a correlation matrix.
+{{< since 2.37 >}}
+
+<!-- matrices; cholesky_factor_corr_jacobian; (vectors y, int K); -->
+\index{{\tt \bfseries cholesky\_factor\_corr\_jacobian }!{\tt (vectors y, int K): matrices}|hyperpage}
+Takes a vector `y` and integer `K`, where `length(y) == choose(K, 2)`, and returns
+a `K` by `K` Cholesky factor of a correlation matrix.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+`matrices` **`cholesky_factor_corr_jacobian`**`(vectors y, int K)`<br>\newline
+
+{{< since 2.37 >}}
+
+<!-- vectors; cholesky_factor_corr_unconstrain; (matrices x); -->
+\index{{\tt \bfseries cholesky\_factor\_corr\_unconstrain }!{\tt (matrices x): vectors}|hyperpage}
+
+`vectors` **`cholesky_factor_corr_unconstrain`**`(matrices x)`<br>\newline
+Takes `x`, a lower triangular square ($K$ by $K$) matrix which is the Cholesky factor of a correlation matrix,
+and returns a corresponding free vector of length $K\choose 2$.
+{{< since 2.37 >}}
+
+### Cholesky factors of covariance matrices
+
+These functions perform the mathematical transform described in the
+[Cholesky Factors of Covariance Matrices section](https://mc-stan.org/docs/reference-manual/transforms.html#cholesky-factors-of-covariance-matrices).
+
+<!-- matrices; cholesky_factor_cov_constrain; (vectors y, int M, int N); -->
+\index{{\tt \bfseries cholesky\_factor\_cov\_constrain }!{\tt (vectors y, int M, int N): matrices}|hyperpage}
+
+`matrices` **`cholesky_factor_cov_constrain`**`(vectors y, int M, int N)`<br>\newline
+Takes a free vector `y` and integers `M` and `N` and returns the `M` by `N` Cholesky
+factor of a covariance matrix. Note that `y` must have length `N + choose(N, 2) + (M - N) * N`,
+and `M` must be greater than or equal to `N`.
+{{< since 2.37 >}}
+
+<!-- matrices; cholesky_factor_cov_jacobian; (vectors y, int M, int N); -->
+\index{{\tt \bfseries cholesky\_factor\_cov\_jacobian }!{\tt (vectors y, int M, int N): matrices}|hyperpage}
+
+`matrices` **`cholesky_factor_cov_jacobian`**`(vectors y, int M, int N)`<br>\newline
+Takes a free vector `y` and integers `M` and `N` and returns the `M` by `N` Cholesky
+factor of a covariance matrix. Note that `y` must have length `N + choose(N, 2) + (M - N) * N`,
+and `M` must be greater than or equal to `N`.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+{{< since 2.37 >}}
+
+<!-- vectors; cholesky_factor_cov_unconstrain; (matrices x); -->
+\index{{\tt \bfseries cholesky\_factor\_cov\_unconstrain }!{\tt (matrices x): vectors}|hyperpage}
+
+`vectors` **`cholesky_factor_cov_unconstrain`**`(matrices x)`<br>\newline
+Takes a $M$ by $N$ Cholesky factor of a covariance matrix `x` and returns
+the corresponding free vector of length $N + N\choose 2 + (M - N)N$.
+
+{{< since 2.37 >}}
+
+### Correlation matrices
+
+These functions perform the mathematical transform described in the
```

Comment:
Same comment here about at least a sketchy definition---a correlation matrix is a positive-definite matrix with a unit diagonal.